### DISCUSSION TITLE
Is work with Next13

### DISCUSSION THREAD

**Discussion by wilonweb** [timlrx/tailwind-nextjs-starter-blog#616] [Q&A]
Hello, i'm learning NextJS 13 ans i would like know if the starter-blog works with the version 13 ? 

Thanks

**Comment by manuva** [comment]
It does not, support needs to be added

**Reply by wilonweb** [reply]
Thanks, 
And do you know if we can see what we have to add 
do you know what we need to add as support ?
If it is written in the documentation ?
And if the project will be updated for version 13 of NextJS ?
